dev/tasks/macros.jinja

```diff
@@ -235,6 +235,7 @@ on:
       brew unlink python@2 || true
       brew config
       brew doctor || true
+      brew install --overwrite python@3.11 python@3.10
```

Comment:
You mentioned in the comments submitting an issue to homebrew — could we link to that here in a comment? Just so we know _why_ we need the overwriting situation?